On hex 0.1.2 zstd.h issue

Had to switch to:

```
      {:bakeware, github: "bake-bake-bake/bakeware", branch: "main"},

```
to get around 
```
cc -c -O2 -Wall -Wextra -Wno-unused-parameter -pedantic -o /app/_build/prod/lib/bakeware/obj/unzstd.o src/unzstd.c
src/unzstd.c:2:10: fatal error: zstd/lib/zstd.h: No such file or directory
    2 | #include "zstd/lib/zstd.h"
      |          ^~~~~~~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:52: /app/_build/prod/lib/bakeware/obj/unzstd.o] Error 1
could not compile dependency :bakeware, "mix compile" failed. You can recompile this dependency with "mix deps.compile bakeware", update it with "mix deps.update bakeware" or clean it with "mix deps.clean bakeware"
==> chira_submit
** (Mix) Could not compile with "make" (exit status: 2).
```

--- a/mix.exs
+++ b/mix.exs
@@ -53,7 +53,8 @@ defmodule Bakeware.MixProject do
         "README.md",
         "CHANGELOG.md",
         "LICENSE",
-        "src/*.[ch]"
+        "src/*.[ch]",
+        "src/zstd"
       ],
       licenses: ["Apache-2.0"],
       links: %{

